packages/client/hmi-client/src/page/data-explorer/components/tera-filter-bar.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-	<div class="flex align-items-center pt-2 pb-2 pl-8 pr-8">
+	<div class="flex align-items-center">
```

Comment:
We don't like these class names right? 😈

Reply:
indeed